test/Altinn.Notifications.Tests/Notifications.Core/TestingServices/OrderProcessingServiceTests.cs

```diff
@@ -175,11 +242,78 @@ public async Task ProcessOrderRetry_SerivceThrowsException_ProcessingStatusIsNot
                 Times.Never);
     }
 
+    [Fact]
+    public async Task IsSendConditionMet_NoConditionEndpoint_ReturnsTrue()
+    {
+        // Arrange
+        NotificationOrder order = new() { Id = Guid.NewGuid(), ConditionEndpoint = null };
+        var orderProcessingService = GetTestService();
+
+        // Act
+        bool actual = await orderProcessingService.IsSendConditionMet(order, isRetry: false);
+
+        // Assert
+        Assert.True(actual);
+    }
+
+    [Theory]
+    [InlineData(true, false)]
+    [InlineData(false, false)]
+    [InlineData(true, true)]
+    [InlineData(false, true)]
+    public async Task IsSendConditionMet_SuccessResultFromClient_ReturnsSameAsClient(bool expectedConditionResult, bool isRetry)
```

Comment:
I'm not sure if the `isRetry` parameter is making a difference. The setup is such that the result match statement will never hit the error case where the `isRetry` value is used.

Reply:
Thought here was to be explicit on expected outcome for all cases, so we are able to catch if behavior of code changes in the future. I didn't set up any other happy path tests (condition is true) for retry.